Docs/Configuration.tex

```diff
@@ -6586,7 +6582,7 @@ \subsection{ProtocolOverrides Properties}\label{uefiprotoprops}
   \texttt{HashServices}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Forcibly reinstalls Hash Services protocols with builtin
+  \textbf{Description}: Forcibly replaces Hash Services protocols with builtin
```

Comment:
Forcibly is really an artefact of some old writing style. The behaviour is all the same. In my opinion, either drop `Forcibly` everywhere, or add it.

Reply:
Will drop remaining examples